media-libs/mesa/mesa-9999.ebuild

```diff
@@ -188,6 +194,12 @@ BDEPEND="
 	opencl? (
 		>=sys-devel/gcc-4.6
 	)
+	rusticl? (
+		>=dev-lang/rust-1.59.0
```

Comment:
You always want virtual/rust, not dev-lang/rust, to allow people to use rust-bin.